RPerformanceTracking/Private/_RPTEnvironment.m

```diff
@@ -56,5 +67,9 @@ -(NSURL*)performanceTrackingBaseURLFromConfig {
     return nil;
 }
 
+- (NSString *)performanceTrackingVersion {
+    NSBundle *bundle = [NSBundle bundleWithIdentifier:@"org.cocoapods.RPerformanceTracking"];
+    return [bundle objectForInfoDictionaryKey:@"CFBundleShortVersionString"] ?: RPTSDKVersion;
```

Comment:
I think RPTSDKVersion is in higher priority than CFBundleShortVersionString, and just RPTSDKVersion is enough.